src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -3351,6 +3356,7 @@ namespace HVACUnitaryBypassVAV {
         int splitterOutNode = CBVAV(CBVAVNum).SplitterOutletAirNode;
         DataLoopNode::Node(splitterOutNode).MassFlowRateSetPoint = DataLoopNode::Node(OutletNode).MassFlowRateSetPoint;
         DataLoopNode::Node(OutletNode) = DataLoopNode::Node(splitterOutNode);
+        DataLoopNode::Node(OutletNode).TempSetPoint = CBVAV(CBVAVNum).OutletTempSetPoint;
```

Comment:
Go ahead and actually set node TempSetPoint since now the CBVAV can be used with OASystems. OutletTempSetPoint was used internally before and now a mixed air SPM could be used.